src/index.js

```diff
@@ -1 +1,4 @@
-import _ from 'lodash';
\ No newline at end of file
+import _ from 'lodash'; /* eslint-disable */
+import './style.css';
+
+element.innerHTML = _.join(['Hello', 'webpack'], ' ');
```

Comment:
- [ ] Nice start but let's make it great, please use the arrow function as the requirements say

![llll](https://user-images.githubusercontent.com/61562626/142073013-a8cc5844-428c-489f-bd95-b10daae21858.jpeg)
.

Reply:
True, but the function that you will add it will not work yet.